client/src/components/Offer/Offer.module.scss

```diff
@@ -0,0 +1,172 @@
+$isNewMarkColor: rgb(230, 73, 0);
+$locationPin: darkorange;
+$dateIcon: rgb(70, 101, 240);
+
+@mixin stylesforDetails {
+  color: rgba(119, 119, 119, 0.664);
+  font-size: 0.8rem;
+  margin: 0.2rem;
+  display: flex;
+  align-items: center;
+}
+
+.isNew {
+  border-left: $isNewMarkColor 3px solid;
+}
+
+.offer {
+  position: relative;
+  background-color: rgb(255, 255, 255);
+  padding: 1rem;
+  margin: 0.5rem;
+  display: flex;
+  flex-direction: row;
+  box-shadow: 0 4px 8px 0 rgb(0 0 0 / 10%), 0 6px 20px 0 rgb(0 0 0 / 5%);
+  border-radius: 2px;
+  color: rgba(0, 0, 0, 0.733);
+  text-decoration: none;
+  transition: 0.3s;
+  max-width: 1000px;
+
+  .newTag {
```

Comment:
It shouldn't push the content: https://www.loom.com/share/b29b7985ef754951aebd9e16b4cb1ae5
Right now the new and not new offers will have strange offsets when placed near.

Reply:
Still valid, but appraently in different file